models/src/main/java/com/vimeo/networking2/params/PublishToFacebookPost.kt

```diff
@@ -13,13 +13,13 @@ data class PublishToFacebookPost(
          * The title of the post as it will appear on Facebook.
          */
         @Json(name = "title")
-        val title: String,
+        val title: String?,
```

Comment:
We should follow the pattern used elsewhere in these param classes and provide default `null` values